client/components/overview/__test__/stylesList.test.js

```diff
@@ -12,3 +13,24 @@ const { queryByTestId } = render(<App />);
 test('Style module should render the list of styles', () => {
   expect(screen.findByTestId('StylesList')).toBeTruthy();
 });
+
+xtest('Should display a default style', () => {
+  expect().toBeTruthy();
+});
+
+xtest('Should update selected style based on user\'s input', () => {
+  expect().toBeTruthy();
+});
+
+describe('Tests using sample data from data dump', () => {
```

Comment:
Remember to either update this or provide dummy data when we start utilizing live data